chat/utils.py

```diff
@@ -4,6 +4,45 @@
 import google.generativeai as genai
 
 import markdown
+import psycopg2
+from psycopg2.extras import RealDictCursor
+
+
+def natural_language_to_sql(question, db_schema):
+    GOOGLE_API_KEY = os.getenv('GOOGLE_API_KEY', "get_one_from_google")
+    genai.configure(api_key=GOOGLE_API_KEY)
+    model = genai.GenerativeModel('gemini-1.5-pro')
+
+    prompt = f"""
+    Given the following database schema:
+    {db_schema}
+
+    Convert the following natural language question into a SQL query:
+    "{question}"
+
+    Return only the SQL query, without any additional explanation.
```

Comment:
See how I set the model instructions in the other code.
That should work much better for enfocing SQL only. 

Prompt needs some work too.